public/css/main.css

```diff
@@ -0,0 +1,934 @@
+/*&&&&&&&&&&&&&&&&&&&&&&&&&&
+
+    General styles
+
+&&&&&&&&&&&&&&&&&&&&&&&&&&&*/
+html {
+  font-family: Helvetica, sans-serif;
+  font-size: 1em;
+}
+body {
+  width: 100%;
+}
+article {
+  clear: left;
+  padding: 2em 2em 1em 2em;
+}
+p {
+  line-height: 1.5;
+  letter-spacing: 0.02em;
```

Comment:
Just a typographical note: Helvetica, especially for body text, should not be letter-spaced. Typography nerds (and most designers are) have a special place in their hearts for Helvetica, and messing with the spacing like this is sacrilege. 

Just something you should be aware of when working with designers. The good ones take type very seriously.